.circleci/config.yml

```diff
@@ -872,7 +872,7 @@ jobs:
       test_command:
         description: Test command to execute (test or coverage)
         type: string
-        default: test
+        default: test --deny-warnings
```

Comment:
Now that I think about it, this will prevent tests from running if there are compiler errors, which prevents the dev from getting useful information in CI. 

An alternative approach could be to add `just build-dev --deny-warnings` at the end of the test workflow. That will extend the full run by about 1 minute, but allows the dev to see if a test is failing sooner. 

A nice in-between might be to put `--deny-warnings on the `--re-run` job?